projects/batfish/src/org/batfish/main/Batfish.java

```diff
@@ -3578,6 +3578,20 @@ public void pushDeltaEnvironment() {
       return configurationData;
    }
 
+   static List<Path> listAllFiles(Path configsPath) {
```

Comment:
add a javadoc comment explaining what this function does. Something simple; you do not need `@param` etc.

```java
/**
 * Returns a list of {@link Path paths} to files under the directory indicated
 * by {@code configsPath}. Directories are recursively expanded but not
 * included in the returned list.
 *
 * <p>Temporary files (with names that start with {@code .}) are omitted from the returned list.
 */
```

is probably what I'd write.